apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/container/edit.js

```diff
@@ -330,41 +339,77 @@ function Edit( props ) {
 		);
 	}
 
-	let stripeNudge = null;
-
-	if ( ! shouldUpgrade && apiState !== API_STATE_CONNECTED && connectURL ) {
-		const stripeConnectUrl = getConnectUrl( props, connectURL );
+	const shouldShowConnectButton = () => {
+		if ( ! shouldUpgrade && apiState !== API_STATE_CONNECTED && connectURL ) {
+			return true;
+		}
 
-		stripeNudge = <StripeNudge { ...props } stripeConnectUrl={ stripeConnectUrl } />;
-	}
+		return false;
+	};
```

Comment:
Nitpick and non-blocking: This can be simplified to

```
const shouldShowConnectButton = () => ! shouldUpgrade && apiState !== API_STATE_CONNECTED && connectURL;
```

Reply:
Thanks, I'll fix this in a followup PR.